pyproject.toml

```diff
@@ -117,6 +118,18 @@ enable_error_code = [
     "truthy-bool",
 ]
 
+[tool.pytest.ini_options]
+minversion = "6.0"
+testpaths = ["tests"]
+log_cli_level = "INFO"
+xfail_strict = true
+filterwarnings = ["error"]
```

Comment:
I thought `filterwarnings = ["error"]` would make all warnings errors. I remember we had many warnings, so I am a bit surprised that the tests pass.

Reply:
I think our pytest.ini overwrites your new pyproject.toml settings https://github.com/scikit-hep/pylhe/blob/main/pytest.ini

Reply:
Yes, we can enable it, but probably should then filter `.*is deprecated and will be removed in a future version.*`. So something like:

```toml
filterwarnings = [
  "error",
  "ignore:.*is deprecated and will be removed in a future version.*:DeprecationWarning"
]
```

Alternatively, rewriting the tests to not test the old deprecated code would reduce the (backwards-compatibility)  coverage.